src/agent.js

```diff
@@ -17,6 +17,11 @@ const botStates = [];
 const botMap = new Map();
 const config = require('../bot-config.json');
 
+let txHandlerCount = 0;
+let blockHandlerCount = 0;
+let gatherMode = 'any';
```

Comment:
Read this value from the `bot-config.json` file.

Reply:
So, this one is interesting. While writing this, I wasn't entirely sure how I wanted this to work.

I think the obvious case is 'any', might even make sense for that to be the default/implicit, and allow the user to set it to any/all explicitly if they'd like.